src/itype.h

```diff
@@ -300,7 +300,8 @@ struct armor_portion_data {
     // how breathable this part of the armor is
     // cached from the material data
     // only tracked for amalgamized body parts entries
-    int breathability = 0; // NOLINT(cata-serialize)
+    // if left the default -1 the value will be recalculated,
+    int breathability = 1; // NOLINT(cata-serialize)
```

Comment:
Is this supposed to be -1?

Reply:
yes, I use it to determine if the item actual breathability is 0 set in JSON as override. So if the value is -1 it will get recalculated as was before my changes

Reply:
```suggestion
    int breathability = -1; // NOLINT(cata-serialize)
```